src/contexts/RoomContext.ts

```diff
@@ -42,6 +42,7 @@ const RoomContext = createContext<IState>({
     canReply: false,
     useIRCLayout: false,
     matrixClientIsReady: false,
+    dragCounter: 0,
```

Comment:
This is probably not necessary here? But there is `draggingFile: false` is that intentional?

Reply:
I'm slightly worried that counting enters/leaves will cause problems and desync (there's a ton of ways to leave a browser window without triggering a dragleave event, making you always +1)

Can we get away with a simple `isDragging` instead?

Reply:
This method is the only solution I've found... I don't think I quite understand what you mean by `isDragging` here. There seems to be a  `isDragging` method in `react-dnd` but I don't think this can be used here. What am I missing?